### DISCUSSION TITLE
How do you save a generated parse tree as a .svg file?

### DISCUSSION THREAD

**Discussion by lihen12** [antlr/intellij-plugin-v4#445] [Q&A] [answered]
As the title states, I can't seem to figure out how to save a parse tree as a .svg file. The only options I get are to "Export to image (white background)" and "Export to image (transparent background)". This produces a .png file only.

**Comment by bjansen** [comment, marked as answer]
On macOS, the IDE uses native file dialogs, which unfortunately doesn't show a combobox with supported file formats. You'll have to manually set the extension to `.svg`.

On Windows, you should see a combobox with supported file formats: 
<img src="https://user-images.githubusercontent.com/281528/52262550-0e32fb00-292d-11e9-9ae3-aa80b8edc538.png"/>

See also #352
